tests/test_image_convert.py

```
@@ -88,6 +89,103 @@ def test_small_frame_size_scales() -> None:
+    [
+        ((1200, 1600), (1600, 1200)),
+        ((2560, 1440), (1440, 2560)),
+    ],
+)
+def test_known_frame_orientations_are_canonicalized(
+    reported: tuple[int, int], native: tuple[int, int]
+) -> None:
+    assert const.canonical_frame_resolution(*reported) == native
+
+
+@pytest.mark.parametrize("width,height", [(1200, 1600), (2560, 1440)])
```

Ruff (0.16.1)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)